app.config.js

```diff
@@ -272,6 +262,18 @@ module.exports = function (config) {
             ],
           },
         ],
+        [
+          'expo-splash-screen',
+          PLATFORM === 'ios'
```

Comment:
@haileyok Worth noting that there are `android` and `ios` properties available in the config plugin (https://docs.expo.dev/versions/latest/sdk/splash-screen/#configurable-properties) - not sure, but maybe you can utilize this in place of the `PLATFORM` check

Reply:
Perf, that makes sense :pinched_fingers:

Reply:
hmm i guess i need to dig at what might be going on here, wondering if you have any context maybe.

seems like when i launch the app in dark mode now the splash is temporarily white instead of the proper color...do you see something obvious with the app config that is messed up?

Reply:
Ah no, it was present regardless of how I configured it (does look like the way you suggested works fine). I'll try again to be sure, but yea it was only on iOS. I suppose the fullscreen flag could be the problem, was hoping that would be "enough" for now, we'll see.